app/styles/ui/_select.scss

```diff
@@ -21,5 +21,6 @@
 
     // Don't include the default padding from textboxish, it's too much
     padding: 0;
+    width: 100%;
```

Comment:
I think this makes sense since `select` elements inside of our `Select` components are supposed to take the whole horizontal space. Does that sound 👍 ?